### PR TITLE
Add custom text to pronoun field on the user profile settings screen.

### PR DESCRIPTION
**Pull request by cmurtaugh**
This PR adds some custom text below the pronoun field on the user profile settings page. 

It will only display the custom text once the user pronoun feature has been activated.

### CODE REVIEW

**Comment by Chris-Thornton-Harvard** [comment]
👍

File: global.js [hunk 1]
```diff
@@ -122,3 +122,32 @@ function initAddSchoolDetail() {
...
+      If you would like to do so, use this section to identify, \
+      change, or delete your chosen personal pronouns. Currently \
+      Canvas requires users to choose from a limited \
+      prepopulated list of personal pronouns. We respect that \
+      language is constantly evolving and members of our community \
+      should be able to be called and describe their identity \
+      however they choose, so we are working to enable that option \
+      within Canvas. To learn more about gender, identity, \
+      or pronouns and to provide feedback, visit: \
+      <a href=\"https://dib.harvard.edu/gender-pronouns\">\
```
Comment (elliottyates) [inline global.js]: Does this automatically get opened in a new window, or would it make sense to add `target="_blank"`?

File: global.js [hunk 2]
```diff
@@ -122,3 +122,32 @@ function initAddSchoolDetail() {
...
+      https://dib.harvard.edu/gender-pronouns</a>.\
+      ");
+
+      $( "#pronouns" ).parent().parent().children("th").css("vertical-align", "top");
```
Comment (elliottyates) [inline global.js:148]: Is there a convenient place to have a look at the DOM? Doesn't look like it's enabled in residential dev, qa, test, beta, or prod at the moment. Thanks!

**Review by elliottyates — COMMENTED** [review]
Awesome.

File: global.js (hunk 1 above)
Comment (cmurtaugh) [reply global.js]: The window behavior wasn't specified and the link appears as it was given to us, but it seems reasonable to me to add a `target="_blank"`.

File: global.js (hunk 2 above)
Comment (cmurtaugh) [reply global.js:148]: Not sure what you mean; what would you want to have enabled?

**Pull request closed without merging**